extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/pages/contract_agreements/ContractAgreementPageApiService.java

```diff
@@ -48,9 +49,20 @@ public ContractAgreementPage contractAgreementPage(DSLContext dsl, @Nullable Con
             return new ContractAgreementPage(cards.toList());
         } else {
             var filtered = cards.filter(card ->
-                card.getTerminationStatus().equals(contractAgreementPageQuery.getTerminationStatus()))
+                    card.getTerminationStatus().equals(contractAgreementPageQuery.getTerminationStatus()))
                 .toList();
```

Comment:
Are we back to 8 space indentation?

Reply:
Seems so, this is correctly auto-formatted